components/Table/src/index.scss

```diff
@@ -99,7 +99,7 @@
 .denhaag-table__body > .denhaag-table__row:hover .denhaag-table__cell,
 .denhaag-table__body > .denhaag-table__row--hover .denhaag-table__heading,
 .denhaag-table__body > .denhaag-table__row:hover .denhaag-table__heading {
-  --denhaag-table-row-background-color: var(--denhaag-table-row-hover-background-color, transparent);
+  --denhaag-table-row-background-color: none;
```

Comment:
Background-color: none is invalid.
<img width="229" alt="Schermafbeelding 2022-10-18 om 14 21 44" src="https://user-images.githubusercontent.com/94894596/196427990-e3185e6f-58ff-4eb4-881a-4f01c905050e.png">

Reply:
Gefixt, zie laatste commit.

Reply:
Token still exists: https://github.com/nl-design-system/denhaag/blob/www.denhaag.nl/proprietary/Components/src/denhaag/table.tokens.json#L52